app/views/govuk_publishing_components/components/docs/subscription-links.yml

```diff
@@ -15,3 +23,17 @@ examples:
   with_only_feed_link:
     data:
       feed_link: '/foreign-travel-advice/singapore.atom'
+  with_custom_text:
+    data:
+      email_signup_link: '/foreign-travel-advice/singapore/email-signup'
+      email_signup_link_text: 'Get notifications'
+      feed_link: '/foreign-travel-advice/singapore.atom'
+      feed_link_text: 'View feed'
+  with_copyable_feed_link:
+    description: |
+      This option changes the feed link to a toggle control, which opens a hidden element containing an input containing the value passed to the component, usually a URL to an atom feed. This uses the [form input](/component-guide/input) component.
```

Comment:
Nitpick: this reads a bit weirdly "..which opens a hidden element **containing** an input **containing** the value...". 

Could we say "..which opens a hidden element **containing** an input **prepopulated with** the value...

Reply:
Much better, thanks.